src/tools/merssh/command.cpp

```diff
@@ -405,3 +402,20 @@ void Command::maybeUndoCMakePathMapping()
         }
     }
 }
+
+void Command::updateOrAddToCMakeCacheIf(QString *data, const QString &name, const QStringList &types,
```

Comment:
Please consider adding updateOrAddToCMakeCacheIf in a separate commit

Reply:
Also, I think the function name is a bit misleading now - it is not really CMakeCache specific

Reply:
yup I struggled to do the refactorization with a separate commit but that wouldn't really make much sense and given the size of the change I chose to keep it in one commit. The function is definitely CMakeCache specific.